Makefile

```diff
@@ -18,11 +18,15 @@ lint: collection-cleanup collection-prep
 # Assumes ansible-test is available in the global scope, such as within the devcontainer environment
 .PHONY: sanity
 sanity: collection-cleanup collection-prep
-	poetry run tests/run-sanity.sh
+	poetry run tests/run-sanity.sh $(filter-out $@,$(MAKECMDGOALS))
```

Comment:
In my first contribution back to this repo in months, I'm going to flip this over to uv 😛